execution/executor-types/src/lib.rs

```diff
@@ -153,6 +155,25 @@ impl StateComputeResult {
             signature: None,
         }
     }
+
+    pub fn new_dummy_with_root_hash(root_hash: HashValue) -> Self {
+        Self {
+            root_hash,
+            frozen_subtree_roots: vec![],
+            num_leaves: 0,
+            parent_frozen_subtree_roots: vec![],
+            parent_num_leaves: 0,
+            epoch_state: None,
+            compute_status: vec![],
+            transaction_info_hashes: vec![],
+            reconfig_events: vec![],
+            signature: None,
+        }
+    }
+
+    pub fn new_dummy() -> Self {
```

Comment:
add some comments how this is used in ordering state computer

Reply:
resolved